server/text_generation_server/models/custom_modeling/t5_modeling.py

```diff
@@ -45,6 +45,15 @@
     SpeculativeHead,
 )
 
+# copied from https://github.com/huggingface/transformers/blob/cd4584e3c809bb9e1392ccd3fe38b40daba5519a/src/transformers/models/t5/modeling_t5.py#L1316
+# Warning message for FutureWarning: head_mask was separated into two input args - head_mask, decoder_head_mask
+__HEAD_MASK_WARNING_MSG = """
```

Comment:
???? Why do we need this ?

Reply:
this is also a `undefined-name` lint, `__HEAD_MASK_WARNING_MSG` is called as a warning in `T5ForConditionalGeneration.forward` however I'm not sure we need this warning... leaving for now but happy to remove